translations/introduction.tex/fr.po

```diff
@@ -70,6 +76,27 @@ msgid ""
 "}\n"
 "\\vfill"
 msgstr ""
+"{2}\n"
+"\\textbf{Notice}: Dans ce livret, les termes et éléments de jeu sont écrits avec leur première lettre en Majuscule.\n"
+"Le \\textbf{Texte en Gras} est utilisé pour attirer votre attention sur des règles importantes.\n"
+"Le \\textit{Texte en Italique} donne des exemples de gameplay.\n"
+"Les \\hyperlink{Heroes of Might and Magic III}{Liens Hypertextes marron} vous emmènent vers d’autres parties de ce livret.\n"
+"% The following glyphs MUST be rendered before their first use in tabular environment in all_map_locations.tex.\n"
+"% Inkscape will fail otherwise.\n"
+"% Other glyphs referenced over there ONLY cause no issues ¯\\_(ツ)_/¯\n"
+"\\phantom{\n"
+"  \\includesvg[height=0.1px]{\\svgs/artifact.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/spellpower.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/movement.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/2_treasure_die.svg}\n"
+"}\n"
+"\\vfill\n"
+"\\columnbreak\n"
+"\\note{10}{Les exceptions et les notes avec des \\hyperlink{Heroes of Might and Magic III}{Liens Hypertextes ambre} sont expliquées dans des cases comme celle-ci.\n"
+"  \\medskip\\\\\n"
+"  Les conflits de règles suivent cet ordre de priorité : Carte du Joueur, Carte d’Unité, Plateau de Ville, Livret de Mission, Livret de Base.\n"
```

Comment:
```suggestion
"  Les conflits de règles suivent cet ordre de priorité: Carte du Joueur, Carte d’Unité, Plateau de Ville, Livret de Mission, Livret de Base.\n"
```

Reply:
In french, a space is required before and after the ":"

Reply:
Didn't know that, thanks!